packages/batch-submitter/src/batch-submitter/tx-batch-submitter.ts

```diff
@@ -742,67 +689,26 @@ export class TransactionBatchSubmitter extends BatchSubmitter {
     this.log.debug('Fetched L2 block', {
       block,
     })
-    const txType = block.transactions[0].txType
 
-    if (this._isSequencerTx(block)) {
-      if (txType === TxType.EIP155 || txType === TxType.EthSign) {
-        return this._getDefaultEcdsaTxBatchElement(block)
-      } else {
-        throw new Error('Unsupported Tx Type!')
-      }
-    } else {
-      return {
-        stateRoot: block.stateRoot,
-        isSequencerTx: false,
-        sequencerTxType: undefined,
-        txData: undefined,
-        timestamp: block.timestamp,
-        blockNumber: block.transactions[0].l1BlockNumber,
-      }
+    const batchElement = {
+      stateRoot: block.stateRoot,
+      timestamp: block.timestamp,
+      blockNumber: block.transactions[0].l1BlockNumber,
+      isSequencerTx: undefined,
```

Comment:
should this be false?
```suggestion
      isSequencerTx: false,
```

Reply:
and is `txData` deprecated?

Reply:
Yep! `txData` and `txType` are both deprecated! 

Also, yes I think it's better for this to be false! Good catch -- updating